### ISSUE TITLE
Feature Request: An efficient way to update WasmEdge's version 

### ISSUE THREAD

**Issue by alabulei1** [WasmEdge/docs#53]
### Is your feature request related to a problem? Please describe.

When WasmEdge has released a new version, we should ensure everything in this doc uses the latest version of WasmEdge.

There are 12 files that need to be synced with the latest version of WasmEdge. It would be complex to maintain all 12 files.

### Describe the solution you'd like.

Can we think of a simple way to update the version of WasmEdge? For example, we can first set a general parameter called `{{ wasmedge_version }}`. Then we update ``{{ wasmedge_version }}` equals the latest version of WasmEdge. Finally, all the docs are using the same and lastest version of WasmEdge.

This would be much easier to update WasmEdge's version. Any feedback is welcome.

**Comment by kokamkarsahil** [comment]
> Can we think of a simple way to update the version of WasmEdge? For example, we can first set a general parameter called `{{ wasmedge_version }}`. Then we update ``{{ wasmedge_version }}` equals the latest version of WasmEdge. Finally, all the docs are using the same and lastest version of WasmEdge.

Yes it's a good approach I was thinking of a build step where it scans all files and replaces that word with a version number specified using sed[0] or other similar tool.

0. https://www.gnu.org/software/sed/manual/sed.html

**Comment by Bhardwaj-Himanshu** [comment]
I do not have any idea about what could solution be, but am happy to help if I could!

**Comment by Sukriti-sood** [comment]
@alabulei1 Are the docs not going to be updated for different versions/releases? If we are, then we can easily change the variable value at any time. Please correct me if I have misunderstood the feature.

**Comment by alabulei1** [comment]
> > Can we think of a simple way to update the version of WasmEdge? For example, we can first set a general parameter called `{{ wasmedge_version }}`. Then we update ``{{ wasmedge_version }}` equals the latest version of WasmEdge. Finally, all the docs are using the same and lastest version of WasmEdge.
> 
> Yes it's a good approach I was thinking of a build step where it scans all files and replaces that word with a version number specified using sed[0] or other similar tool.
> 
> 0. https://www.gnu.org/software/sed/manual/sed.html

This sounds good! A tool like this can reduce human errors

**Comment by alabulei1** [comment]
> @alabulei1 Are the docs not going to be updated for different versions/releases? If we are, then we can easily change the variable value at any time. Please correct me if I have misunderstood the feature.

@hydai @juntao What do you think?

**Comment by Bhardwaj-Himanshu** [comment]
Yup something like-
![image](https://user-images.githubusercontent.com/74407676/236621961-c915bd78-9ec2-47a8-bee6-ba4411446ddc.png)
Could be very useful for onboarding developers!

**Comment by hydai** [comment]
> > @alabulei1 Are the docs not going to be updated for different versions/releases? If we are, then we can easily change the variable value at any time. Please correct me if I have misunderstood the feature.
> 
> @hydai @juntao What do you think? I think we use this way when 1.0 is released.

The 0.x series changes API a lot between versions, and it would be better to have it right now. As you can see [here](https://github.com/WasmEdge/WasmEdge/tree/master/docs/book/en/src/sdk/c), we already split the API reference during different versions.

**Comment by kokamkarsahil** [comment]
Just to add on:
![image](https://user-images.githubusercontent.com/84327775/236659405-219029e3-37b7-4adc-bc27-dbf07caffcaf.png)
Source: https://docusaurus.io/docs/versioning

>The 0.x series changes API a lot between versions, and it would be better to have it right now. As you can see [here](https://github.com/WasmEdge/WasmEdge/tree/master/docs/book/en/src/sdk/c), we already split the API reference during different versions.

We can tag this version for now and set `disableVersioning` and enable it in future.

**Comment by alabulei1** [comment]
I find how to pass env var in docusarus V2. This will fix this issue faster. https://stackoverflow.com/questions/61127010/how-to-pass-env-var-to-docusaurus-v2

**Comment by alabulei1** [comment]
@flows_summarize

**Comment by alabulei1** [comment]
@flows_summarize

**Comment by alabulei1** [comment]
@flows_summarize
